### ISSUE TITLE
Gpu related error

### ISSUE THREAD

**Issue by Sri100603** [cypress-io/cypress#30319]
### Current behavior
```
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null) [3328146:0930/123615.535963:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
````

### Desired behavior

when `npx cypress` is run, it should run all the spec files.

### Test code to reproduce

`npx cypress` is resulting in issue

### Cypress Version

10.11.0

### Node version

20

### Operating System

Linux  Redhat 8.10

### Debug Logs

```shell
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[3328146:0930/123615.535963:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
Couldn't find tsconfig.json. tsconfig-paths will be skipped
```

### Other

<img width="993" alt="Screenshot 2024-09-30 182631" src="https://github.com/user-attachments/assets/e54bab16-151b-4a83-9ed6-0f22c316c55a">

**Comment by MikeMcC399** [comment]
@Sri100603 

- See issue https://github.com/cypress-io/cypress/issues/25357 which was resolved in Cypress `13.2.0`

You should consider updating to the latest Cypress `13.x` version. There is a [Migration Guide](https://docs.cypress.io/guides/references/migration-guide) available to help with the changes.

**Comment by jennifer-shehane** [comment]
Closing as resolved.
